integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java

```diff
@@ -103,136 +129,276 @@
   private static LoggingFacade logger = null;
 
   /**
-   * Assigns unique namespaces for operator and domains. Pull WebLogic image if running tests in Kind cluster. Installs
-   * operator.
+   * Assigns unique namespaces for operator and domains.
+   * Pull WebLogic image if running tests in Kind cluster.
+   * Installs operator.
    *
    * @param namespaces injected by JUnit
    */
   @BeforeAll
-  public static void initAll(@Namespaces(2) List<String> namespaces) {
+  public static void initAll(@Namespaces(3) List<String> namespaces) {
     logger = getLogger();
     logger.info("Assign a unique namespace for operator");
     assertNotNull(namespaces.get(0), "Namespace is null");
     opNamespace = namespaces.get(0);
     logger.info("Assign a unique namespace for WebLogic domain");
     assertNotNull(namespaces.get(1), "Namespace is null");
-    domainNamespace = namespaces.get(1);
+    domainNamespace1 = namespaces.get(1);
+    assertNotNull(namespaces.get(2), "Namespace is null");
+    domainNamespace2 = namespaces.get(2);
+
+    // set the service account name for the operator
+    opServiceAccount = opNamespace + "-sa";
+
+    // install and verify operator with REST API
+    installAndVerifyOperator(opNamespace, opServiceAccount, true, 0, domainNamespace1);
+    externalRestHttpsPort = getServiceNodePort(opNamespace, "external-weblogic-operator-svc");
 
-    // install operator and verify its running in ready state
-    installAndVerifyOperator(opNamespace, domainNamespace);
     // create pull secrets for WebLogic image when running in non Kind Kubernetes cluster
     // this secret is used only for non-kind cluster
-    createSecretForBaseImages(domainNamespace);
-
+    createSecretForBaseImages(domainNamespace1);
   }
 
   /**
    * Test domain events are logged when domain resource goes through various life cycle stages.
    * Verifies DomainCreated is logged when domain resource is created.
    * Verifies DomainProcessingStarting is logged when operator processes the domain resource.
    * Verifies DomainProcessingCompleted is logged when operator done processes the domain resource.
-   * Verifies DomainChanged is logged when operator processes the domain resource changes.
-   * Verifies DomainProcessingRetrying is logged when operator retries the failed domain resource changes.
-   * Verifies DomainProcessingAborted is logged when operator exceeds the maximum retries.
    */
+  @Order(1)
   @Test
-  @DisplayName("Test domain events for various domain life cycle changes")
-  public void testDomainK8SEvents() {
-    DateTime timestamp = new DateTime(System.currentTimeMillis());
+  @DisplayName("Test domain events for various successful domain life cycle changes")
+  public void testDomainK8SEventsSuccess() {
+    DateTime timestamp = new DateTime(Instant.now().getEpochSecond() * 1000L);
```

Comment:
This seems complicated. Isn't there a DateTime.now()?

Reply:
The returned events list by the Kubernetes client api seems to be rounded off to the nearest second hence we avoided using DateTime.now()